src/coreclr/debug/daccess/request.cpp

```diff
@@ -3613,14 +3613,18 @@ ClrDataAccess::TraverseVirtCallStubHeap(CLRDATA_ADDRESS pAppDomain, VCSHeapType
                 break;
 
             case CacheEntryHeap:
+#ifdef FEATURE_VIRTUAL_STUB_DISPATCH
                 pLoaderHeap = pVcsMgr->cache_entry_heap;
+#else
+                hr = S_OK;
```

Comment:
Is this needed? The hr is always initialized to S_OK in the SOSDacEnter macro and the code here already relied on that.

Reply:
No it isn't. I'll get rid of this.